.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Question
```

Comment:
We don't want actual question issues, do we?
For that the GitHub discussions are the appropriate place.
So I'd suggest completely replacing this issue template by three contact links.

For an example look at the Gradle repo,
Here the config: https://github.com/gradle/gradle/blob/master/.github/ISSUE_TEMPLATE/config.yml
Here the result: https://github.com/gradle/gradle/issues/new/choose